backend.py

```diff
@@ -129,7 +129,12 @@ def find_between(s, first, last):
             text = text.replace("\n", "\\n")
             lines.extend([f'canvas.create_text({x},{y},text="{text}",fill="{color}",font=("{font}",int({fontSize})))\n'])
 
-        elif element["name"] == 'TextBox':
+        elif element["name"] in ('TextBox', 'TextArea'):
```

Comment:
The changes here simply differentiate between an element named `TextBox` and an element named `TextArea`. I ran some basic tests to check for compatibility, and didn't run into any issues. The elements seem the same in every way except the `Text` element allows multi-line input.

Reply:
I'll try it once and merge it once done. Thanks for the addition.

Reply:
Also wouldn't it be nice if we directly name it TextArea ? I think Naming it text could be confusing.

Reply:
Do you mean making a separate `elif` for the `Text` object? I think that comes down to preference, so it's whatever you think works best! I figured that since the elements in Tkinter were so similar, it wouldn't hurt to spare some duplicate lines of code lol

Reply:
Ohh, Sorry. I misunderstood the code. Got it.

Reply:
I tested the code with different use cases and it works fine. I think there wont be a problem merging it into main branch.